processor/src/main/java/com/blarley/rxretrobusseed/annotationprocessor/processor/RxRetroBusAnnotationProcessor.java

```diff
@@ -71,7 +71,7 @@ public boolean process(Set<? extends TypeElement> annotations, RoundEnvironment
             for (Element subElement : roundEnv.getElementsAnnotatedWith(Publish.class)) {
 
                 // ExecutableElements represent methods (among other things) - TODO: Figure out how this can break
-                if (subElement instanceof ExecutableElement) {
+                if (subElement instanceof ExecutableElement && subElement.getEnclosingElement() == element) {
```

Comment:
Filters out methods annotated with `@Publish` in other elements.

`roundEnv.getElementsAnnotatedWith(Publish.class)` retrieves **ALL** elements annotated with `@Publish`, which leads to a compilation error when multiple interfaces are present. We only want the methods annotated with `@Publish` inside the scope of the given element.